app/views/stories/edit.html.erb

```diff
@@ -0,0 +1,69 @@
+<h1>Editing Story</h1>
+
+<%= form_with(model:@story, local:false) do |form| %>
+
```

Comment:
almost all of this form is being duplicated here. you should DRY this.

Reply:
Marvelously done, Thanks